huggingface/pytorch/buildspec-1-10.yml

```diff
@@ -0,0 +1,32 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
I think we should rename it to `buildspec-trcomp-1-10.yml`. Can you please look in folder: https://github.com/aws/deep-learning-containers/tree/master/huggingface/pytorch and see the naming convention that is followed for the old trcomp buildspecs ?